node/actors/internal_nats_server.go

```diff
@@ -62,9 +69,9 @@ func (ns *internalNatsServer) HandleMessage(from gen.PID, message any) error {
 			return err
 		}
 		ns.tokens[InternalNatsServerReadyName] = token
-		ns.Log().Info("registered publishable event %s, waiting for consumers...", InternalNatsServerReadyName)
+		ns.Log().Info("registered publishable event, waiting for consumers...", slog.Any("atom", InternalNatsServerReadyName))
```

Comment:
the field name here isn't `atom`, it's the name of a published event, so call it `event_name`

Reply:
Copypasta artifact.